### PR TITLE
Use primaryMediaType from the passport taggings to determine active nav item (instead of when media is found in the page data)

### PR DESCRIPTION
**Pull request by LilyL0u**
https://bbc-tpg.slack.com/archives/C03S8UQJZ6U/p1789374832403519

This pull request refactors how the primary media type (audio or video) is determined for a page, making it rely exclusively on explicit passport taggings rather than inferring from block types or content blocks. It also introduces a new utility function for this purpose and updates related tests accordingly.

**Primary Media Type Determination Refactor:**

* Replaces the logic in `PageLayoutWrapper` to determine `primaryMediaType` using the new `getPrimaryMediaType` utility, which inspects `passport` taggings in metadata, instead of scanning `blockTypes` or content blocks. Now, only explicit passport taggings for primary media type are considered (`src/app/components/PageLayoutWrapper/index.tsx`). [[1]](diffhunk://#diff-b60d5f8fba0b8f10798660be71a5e9a35d38550578d757f78603adf6f0c2bfa8R5-R7) [[2]](diffhunk://#diff-b60d5f8fba0b8f10798660be71a5e9a35d38550578d757f78603adf6f0c2bfa8R37-R39) [[3]](diffhunk://#diff-b60d5f8fba0b8f10798660be71a5e9a35d38550578d757f78603adf6f0c2bfa8L59-R66)

* Adds the new utility function `getPrimaryMediaType` that checks for known audio/video thing IDs in passport taggings and returns the corresponding media type or `undefined` (`src/app/lib/utilities/getPrimaryMediaType/index.ts`).

**Testing Updates:**

* Updates and expands tests in `primaryMediaType.test.tsx` to reflect the new logic: tests now verify that only explicit passport taggings set the `primaryMediaType`, and that block types or content blocks alone do not (`src/app/components/PageLayoutWrapper/primaryMediaType.test.tsx`). [[1]](diffhunk://#diff-d553091951514e5597b760fb7bd0239fd4e6a7cdb7f9e21f5db21f6117c906d6R15-R37) [[2]](diffhunk://#diff-d553091951514e5597b760fb7bd0239fd4e6a7cdb7f9e21f5db21f6117c906d6L34-R53) [[3]](diffhunk://#diff-d553091951514e5597b760fb7bd0239fd4e6a7cdb7f9e21f5db21f6117c906d6L48-L96)

* Adds comprehensive unit tests for the new `getPrimaryMediaType` utility, covering all edge cases such as missing, unknown, or absent taggings (`src/app/lib/utilities/getPrimaryMediaType/index.test.ts`).

**Other:**

* Minor cleanup: removes outdated comments and unused logic from navigation and layout components (`src/app/components/Navigation/index.tsx`).Resolves JIRA: 

Summary
======
_A very high-level summary of easily-reproducible changes that can be understood by non-devs, and why these changes where made._

Code changes
======
- _List key code changes that have been made._

Testing
======
1. _List the steps required to test this PR._

## Useful Links
 - [Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)
 - [Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
- _Add Links to useful resources related to this PR if applicable._

### CODE REVIEW

File: src/app/lib/utilities/getPrimaryMediaType/index.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { MetadataTaggings } from '#app/models/types/metadata';
+
+const AUDIO_THING_ID = 'fe1fbc8a-bb44-4bf8-8b12-52e58c6345a4';
+const VIDEO_THING_ID = 'ffc98bca-8cff-4ee6-9beb-a6ff6ef3ef9f';
```
[the remark this answers is not in the document]
Comment (LilyL0u) [reply src/app/lib/utilities/getPrimaryMediaType/index.ts:4]: I don't know if we want the article-sfv as fallback. I think we can leave this for now.

**Review by louisearchibald — APPROVED** [review]

**Review by pvaliani — APPROVED** [review]
Looks good! Just checked in le browser
